assets/.babelrc

```diff
@@ -1,5 +1,17 @@
 {
-    "presets": [
-        "@babel/preset-env"
-    ]
+  "presets": ["@babel/preset-env"],
+  "env": {
```

Comment:
I added this so we don't need `@babel/polyfill` anymore since node supports all the features we're using.